snmp/tests/test_profiles.py

```diff
@@ -2309,10 +2309,10 @@ def test_isilon(aggregator):
 
 
 @pytest.mark.usefixtures("dd_environment")
-def test_apc_ups(aggregator):
-    run_profile_check('apc_ups')
+def test_apc(aggregator):
+    run_profile_check('apc')
     profile_tags = [
-        'snmp_profile:apc_ups',
+        'snmp_profile:apc',
```

Comment:
Shall we keep testing the old names to assert that it's still working?

Maybe we can create 2 test `test_apc_ups()` and `test_apc()` then call `assert_apc(profile_name)`.